lmdeploy/pytorch/engine/engine_instance.py

```diff
@@ -93,9 +93,22 @@ class EngineInstance:
     """
 
     def __init__(self, engine: Engine):
+
+        def __get_max_input_len(engine):
+            """get max input len."""
+            cache_config = engine.cache_config
+            max_input_len = (cache_config.block_size *
+                             cache_config.num_gpu_blocks)
+            window_size = cache_config.window_size
+            if window_size > 0 and window_size <= max_input_len:
+                max_input_len = (1 << 63) - 1
```

Comment:
why give such a large value to `max_input_len`?

Reply:
slide window attention has constant memory usage, there is no need to limit input length.